seed/static/seed/partials/column_mappings.html

```diff
@@ -31,13 +31,13 @@ <h2><i class="fa fa-sitemap"></i> <span translate>Column Mappings</span></h2>
           <span>{$:: 'Column Mapping Preset' | translate $}: </span>
```

Comment:
Not exactly sure where, but changes would likely involve this file...
We should disable the units choice too (for Gross Floor Area right now, but the fix should be for any "unit-aware" field).

![Screen Shot 2020-05-13 at 2 26 24 PM](https://user-images.githubusercontent.com/30608004/81861810-0f86a680-9526-11ea-8e92-f2b4629965b4.png)